### DISCUSSION TITLE
Add addresses to shops that don't have one

### DISCUSSION THREAD

**Discussion by fabiobettani** [streetcomplete/StreetComplete#6024] [Ideas]
I've been mapping shops in the city centre (Bologna, Italy), and I'm missing the ability to add addresses to those that don't have one, or to those that I newly create.

I would love to be able to add addr:street and addr:housenumber via StreetComplete. Am I missing something, or this is not available in SC as of now? Is there a reason that makes this impractical to ask for addresses of shops?

At least in Italy, it's something that can easily be done from the street, as pretty much all shops show their house number right at the entrance.

Especially in historical centres, where shops can be very close to each other, the exact address makes it easier to find them; most of all, if somebody marks the shop as vacant (including through StreetComplete), knowing the exact address is the only way to actually be able to answer the subsequent "This shop has been vacant. What’s here now?" quest.

**Comment by matkoniecz** [comment]
> Is there a reason that makes this impractical to ask for addresses of shops?

there is no need to duplicate address info on shops*, you can put address on a building or on a node within building, and StreetComplete supports this with an address overlay and address quests

address quests in Italy are disabled, see #714

it was discussed few years ago at https://lists.openstreetmap.org/pipermail/talk-it/2018-July/063712.html - has situation changed since then?

*though maybe Italy is special? There is something unusual with addresses there that caused house number quest to be stopped there
